substrate/frame/staking/src/lib.rs

```diff
@@ -404,7 +404,9 @@ pub enum RewardDestination<AccountId> {
 	Staked,
 	/// Pay into the stash account, not increasing the amount at stake.
 	Stash,
-	/// Pay into the controller account.
+	#[deprecated(
+		note = "Controller accounts are deprecated. This variant now behaves the same as `Stash` variant"
```

Comment:
Let's specify a removal date, otherwise this deprecated code is likely to stick around forever. We usually set 3 months in the future.

Reply:
Good call @liamaharon. Completely removing `Controller` has been explored in #1196, hopefully in 3 months we'll be in a better place to merge a similar solution.